simapp/app.go

```diff
@@ -344,7 +344,7 @@ func NewSimApp(
 		appCodec, legacyAmino, app.StakingKeeper, authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 	)
 
-	app.FeeGrantKeeper = feegrantkeeper.NewKeeper(appCodec, runtime.NewKVStoreService(keys[feegrant.StoreKey]), app.AuthKeeper)
+	app.FeeGrantKeeper = feegrantkeeper.NewKeeper(runtime.NewEnvironment(runtime.NewKVStoreService(keys[feegrant.StoreKey]), logger), appCodec, app.AuthKeeper)
```

Comment:
should `appCodec` remain in the same place for consistency

Reply:
i started moving it in other modules as well. I think we can get rid of appcodec here as well. and put it in env